website/src/HomePage.tsx

```diff
@@ -141,7 +138,42 @@ export default function HomePage() {
               </ExperimentalLink>,
             ])}
           </p>
-          <SupportedNetworks className="mt-8" />
+          <div className="graph-docs-not-markdown mt-8 overflow-clip rounded-8 border border-space-1500">
+            <ul className="grid grid-cols-auto-fill-16 gap-px text-space-500">
+              {supportedNetworks
+                // TODO: Don't filter out testnets that don't have a mainnet
+                .filter((network) => network.networkType === NetworkType.Mainnet)
+                // Filter out networks that are either duplicates (same logo, same or similar short name) or irrelevant in this view
+                .filter(
+                  (network) =>
+                    !network.caip2Id.startsWith('beacon:') &&
+                    !['boba-bnb', 'eos-evm', 'polygon-zkevm', 'solana-accounts'].includes(network.id),
+                )
+                .filter((network) => {
+                  return (
+                    // TODO: Fix Zora mono logo in web3icons
```

Comment:
this cannot be "fixed" unless zora decides to gives us a mono logo, we should just delete this todo imo

Reply:
Done in https://github.com/graphprotocol/docs/pull/922/commits/f4e286f995a4a8fd76d49d458379d9176280eba8, thank you!